evap/staff/tools.py

```diff
@@ -65,6 +67,22 @@ def create_user_list_html_string_for_message(users):
     return format_html_join("", "<br />{} {} ({})", ((user.first_name, user.last_name, user.email) for user in users))
 
 
+def append_user_list_if_not_empty(message: str, user_profiles: Iterable[UserProfile]) -> SafeString:
+    message = conditional_escape(message)
+    if not user_profiles:
+        return message + escape(".")
+    return message + escape(": ") + create_user_list_html_string_for_message(user_profiles)
+
+
+def conditional_escape(s: str) -> SafeString:
+    """
+    Like Django's `conditional_escape`, but only handles `str` and `SafeString`.
+    """
```

Comment:
I don't get it. Should it just say "`str`" instead of "`str` and `SafeString`"?

Reply:
Ah, the wording could be more precise to better suggest what the difference is (django also wants to be compatible with other types that claim to be safe). I suppose it could be "distincts" instead of "handles" or so. Or another sentence, don't care